plugins/virtual_event/services/VirtualEventService.php

```diff
@@ -143,33 +147,34 @@ public function listAction(KalturaVirtualEventFilter $filter = null, KalturaFilt
 	
 	protected function validateScheduleEvents (KalturaVirtualEvent $virtualEvent)
 	{
+		$partnerId = kCurrentContext::getCurrentPartnerId();
 		if (!is_null($virtualEvent->agendaScheduleEventId))
 		{
-			$this->getScheduleEvent($virtualEvent->agendaScheduleEventId, VirtualScheduleEventSubType::AGENDA);
+			ScheduleEventPeer::retrieveByPartnerIdAndId($partnerId, $virtualEvent->agendaScheduleEventId, VirtualScheduleEventSubType::AGENDA);
 		}
 		if (!is_null($virtualEvent->registrationScheduleEventId))
 		{
-			$this->getScheduleEvent($virtualEvent->registrationScheduleEventId, VirtualScheduleEventSubType::REGISTRATION);
+			ScheduleEventPeer::retrieveByPartnerIdAndId($partnerId, $virtualEvent->registrationScheduleEventId, VirtualScheduleEventSubType::REGISTRATION);
 		}
 		if (!is_null($virtualEvent->mainEventScheduleEventId))
 		{
-			$this->getScheduleEvent($virtualEvent->mainEventScheduleEventId, VirtualScheduleEventSubType::MAIN_EVENT);
+			ScheduleEventPeer::retrieveByPartnerIdAndId($partnerId, $virtualEvent->mainEventScheduleEventId, VirtualScheduleEventSubType::MAIN_EVENT);
 		}
 	}
 	
 	protected function validateUserGroups (KalturaVirtualEvent $virtualEvent)
```

Comment:
naming is not consistent call the function validateGroups and inner function isValidGroup